src/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -48,13 +48,40 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = "Reads a SAM or BAM file and writes a file containing metrics about " +
-                "the statistical distribution of insert size (excluding duplicates) " +
-                "and generates a Histogram plot.",
-        usageShort = "Writes insert size distribution metrics for a SAM or BAM file",
+        usage = CollectInsertSizeMetrics.USAGE_SUMMARY + CollectInsertSizeMetrics.USAGE_BRIEF,
+        usageShort = CollectInsertSizeMetrics.USAGE_BRIEF,
         programGroup = Metrics.class
 )
 public class CollectInsertSizeMetrics extends SinglePassSamProgram {
+    static final String USAGE_BRIEF = "Metrics about the insert size distribution of a paired-end library";
+    static final String USAGE_SUMMARY = "Tool provides useful metrics for validating library construction including " +
+            "the insert size distribution and read orientation of paired-end libraries. <br /><br />" +
+            "" +
+            "The insert size distribution is output as both a histogram \".insert_size_Histogram.pdf\" and as a data table " +
+            "\".insert_size_metrics.txt\".  In addition, The tool groups reads into three" +
+            " orientation categories: forward reverse (FR), reverse forward (RF), and reverse-reverse/forward-forward (TANDEM). " +
+            "In general, paired-end reads tend" +
```

Comment:
Would be useful to provide an additional note to explicitly clarify the mate pair vs pair-end thing. E.g. say something like:

The expected proportions of these metrics vary depending on the type of library preparation that was used, because there are important technical differences between pair-end libraries and mate-pair libraries. For pair-end sequencing, the library preparation yields a set of fragments, and the machine then sequences each fragment from both ends (for example if you have a 500bp fragment, the machine will sequence e.g. bases 1-75 and bases 425-500 of the fragment). In mate-pair sequencing, the library preparation yields a set of pairs of fragment ends (that are identified as mates), and the machine then performs normal sequencing (from a single side) on each end fragment.